### DISCUSSION TITLE
Plugin request: color rows by label

### DISCUSSION THREAD

**Discussion by jerryfreak** [Novik/ruTorrent#2504] [Ideas]
possible to do this? would make sorting a bit easier

TIA!

**Comment by stickz** [comment]
This would be hard to implement with different themes. It seems sufficient to be able to select groups of torrents by label. `v4.1` has multiple label selection as well.

**Comment by jerryfreak** [comment]
oh didnt realize 4.1 was out. whats the easiest upgrade from 4.0 to 4.1 without losing settings? installed via swizzin
